packages/rule-components/README.md

```diff
@@ -6,24 +6,119 @@ If you want to use rules table component please pass entire dataset from server
 import React, { Component } from 'react';
 import ReactDOM from 'react-dom';
 import './index.scss';
-import RuleTable, { severityLabels } from '../../rule-components/src/RuleTable';
+import RuleTable, { severityLabels } from '@redhat-cloud-services/rule-components/dist/cjs/RuleTable';
```

Comment:
@bond95 having absolute exports is useful for bundle size optimizations. If you import some function, component, constant, etc, from index.js in the non-ESM environment, tree shaking won't be applied and your bundle will end up will every single exported module instead of the one you are using.

Reply:
I understand that, but point of this example is not to reduce bundle size but show how to use this component, newbies will not just understand this import. So my suggestion is use typical import, also some remark about reducing size can be added in README

Reply:
Actually "newbies" should peferably use absolute paths because they probably don't know how to optimize the bundle sizes.

Reply:
I'm agree, but the problem that newbies are don't know what is this and how and why to use it. So, I said, better add to README some additional suggestion like:
```Markdown
If you don't need to use whole package, please, import separate component like this:
 - RuleTable - @redhat-cloud-services/rule-components/dist/cjs/RuleTable
 - RuleFilters - @redhat-cloud-services/rule-components/dist/cjs/RuleFilters
```

Reply:
As @Hyperkid123 said, this is to teach programmers to use direct imports as they are good with tree shaking. I'm going to add note that you can use package import, but be cautious and check if your bundle does bloat or not.

Reply:
I know, but my point was that in case of using direct import somewhere at least should be described why to use import in this way and what’s alternatives.